docs/src/extend/selectors.md

```diff
@@ -46,7 +46,9 @@ The following selectors are supported:
 * following sibling: `VariableDeclaration ~ VariableDeclaration`
 * adjacent sibling: `ArrayExpression > Literal + SpreadElement`
 * negation: `:not(ForStatement)`
+* has: `:has(ForStatement)`
```

Comment:
Eslint doesn't officially support `:has()`, please see the discussions in https://github.com/eslint/eslint/pull/14674 and https://github.com/eslint/eslint/issues/14076 for reasons.

Reply:
My bad 😔. Not sure why I didn't find this when searching. Thanks for the quick review though! I will close now...